core/src/main/java/io/ultrabrew/metrics/data/CursorEntry.java

```diff
@@ -11,7 +11,7 @@
    *
    * @return identifier of the metric
    */
-  String getMetricId();
+  String getMetricName();
```

Comment:
This changes ABI, breaks all existing users that have code dealing with this class, do we _really_ want to go there with this change?

Reply:
That's right. The change to `CursorEntry` would break the reporters. May be I will revert this class.

Reply:
hmm.. yeah i was about to mention that as well. we cannot change anything that's part of public api. @smrutilal2 i guess let's revert it for now.

Reply:
@arungupta and @lonemeow, I have reverted `CursorEntry`. Please take a look.